src/VecSim/algorithms/hnsw/serialization.cpp

```diff
@@ -235,22 +262,30 @@ HNSWIndexSerializer::HNSWIndexSerializer(std::shared_ptr<HierarchicalNSW<float>>
     hnsw_index = std::move(hnsw_index_); // Hold a reference of the index.
 }
 
-void HNSWIndexSerializer::saveIndex(const std::string &location) {
+void HNSWIndexSerializer::saveIndex_v1(const std::string &location) {
```

Comment:
Do we have `saveIndex_v0` or not anymore?

Reply:
We don't need, since from now on we only want to save in V1 (while we do want to enable *loading* of older version...)